### PR TITLE
i18n(de): update translation guides/css-and-tailwind.mdx file into German

### PR DESCRIPTION
**Pull request by trueberryless**
#### What kind of changes does this PR include?

- translate outdated file `guides/css-and-tailwind.mdx` into German

#Hacktoberfest

### CODE REVIEW

File: docs/src/content/docs/de/guides/css-and-tailwind.mdx [hunk 1]
```diff
@@ -181,14 +181,14 @@ Wenn du bereits eine Starlight-Website hast und Tailwind CSS hinzufügen möchte
...
     ```
 
-5.  Füge das Starlight Tailwind-Plugin zu `tailwind.config.cjs` hinzu:
+5.  Füge das Starlight Tailwind-Plugin zu `tailwind.config.mjs` hinzu:
 
     ```js ins={2,7}
-    // tailwind.config.cjs
-    const starlightPlugin = require('@astrojs/starlight-tailwind');
+    // tailwind.config.mjs
+    import starlightPlugin from'@astrojs/starlight-tailwind';
```
Comment (HiDeoo) [inline docs/src/content/docs/de/guides/css-and-tailwind.mdx:188]:
```suggestion
    import starlightPlugin from '@astrojs/starlight-tailwind';
```

File: docs/src/content/docs/de/guides/css-and-tailwind.mdx [hunk 2]
```diff
@@ -181,14 +181,14 @@ Wenn du bereits eine Starlight-Website hast und Tailwind CSS hinzufügen möchte
...
 
     /** @type {import('tailwindcss').Config} */
-    module.exports = {
+    export default = {
```
Comment (HiDeoo) [inline docs/src/content/docs/de/guides/css-and-tailwind.mdx:191]:
```suggestion
    export default {
```

File: docs/src/content/docs/de/guides/css-and-tailwind.mdx [hunk 3]
```diff
@@ -206,12 +206,12 @@ Falls gesetzt, überschreiben die folgenden Optionen die Standard-Styles von Sta
 - `fontFamily.mono` - wird für Code-Beispiele verwendet
 
 ```js {12,14,18,20}
-// tailwind.config.cjs
-const starlightPlugin = require('@astrojs/starlight-tailwind');
-const colors = require('tailwindcss/colors');
+// tailwind.config.mjs
+import starlightPlugin from'@astrojs/starlight-tailwind';
```
Comment (HiDeoo) [inline docs/src/content/docs/de/guides/css-and-tailwind.mdx:210]:
```suggestion
import starlightPlugin from '@astrojs/starlight-tailwind';
```

File: docs/src/content/docs/de/guides/css-and-tailwind.mdx [hunk 4]
```diff
@@ -206,12 +206,12 @@ Falls gesetzt, überschreiben die folgenden Optionen die Standard-Styles von Sta
...
+import colors from 'tailwindcss/colors';
 
 /** @type {import('tailwindcss').Config} */
-module.exports = {
+export default = {
```
Comment (HiDeoo) [inline docs/src/content/docs/de/guides/css-and-tailwind.mdx:214]:
```suggestion
export default {
```

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the update.

Left a few suggestions but I think after these changes this should be perfect 👍

**Comment by trueberryless** [comment]
Okay, admittedly, these "mistakes" were kinda lost by me, but nonetheless, glad you spot'em! Thx, @HiDeoo!

**Review by delucis — APPROVED** [review]
Nice work team! Great review and PR combo 🤩

**Pull request merged**